grid/crm/lists/profile.supr

```diff
@@ -0,0 +1,96 @@
+"""
+Lists
+Manage contact lists in Customer Relation Management systems.
+"""
+
+name = "crm/lists"
```

Comment:
question: What are the benefits of single `crm/lists` profile compared to `crm/get-lists` and `crm/add-contacts-to-lists` profiles ?

 I am not big fan of profiles build around entities. It is hard to harmonise such profiles. All providers needs to implement all of the use cases inside the profile. Comlink by its nature gives us the option to not implement profile by provider.

Reply:
It's a more complete (application) profile - it's hard to add to a list whose ID I cannot obtain. 

But that's a deeper and more general issue, if the convention is to go with per-use case profiles, let's do that for now

Reply:
> it's hard to add to a list whose ID I cannot obtain

I understand, you are saying that it doesn't make sense to have one use without the other.

Reply:
Let's keep it then. I do not know CRM domain properly and do not know the demands.